### PR TITLE
Review Page standalone fields

### PR DESCRIPTION
**Pull request by stephanieharrison-maximus**
### Please check if the PR fulfills these requirements:

- [X] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [X] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
--Hide spouse section entirely if there's no spouse
--Add missing field data
--Move a bunch of stuff into conditionals, eg. only display X field if user is applying for MSP, etc.
--Minor formatting changes

### CODE REVIEW

File: jha/src/components/enrolment/ReviewTableList.vue [hunk 1]
```diff
@@ -761,117 +858,118 @@ export default {
...
-          label: 'Phone',
+        items.push({
+          label: "Phone",
           value: this.$store.state.enrolmentModule.phone,
         });
-      } 
+      }
       return items;
     },
+    getIfHasSpouse() {
```
Comment (bryce-mcmath) [inline jha/src/components/enrolment/ReviewTableList.vue:970]: Could you change this to just "hasSpouse"? Would be clearer

Comment (harrymaynard-maximus) [reply jha/src/components/enrolment/ReviewTableList.vue:970]: I second this. Otherwise looks good.

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good.

**Review by bryce-mcmath — APPROVED** [review]
Looks good!

**Pull request merged**